diff --git a/librarySDK b/librarySDK
--- a/librarySDK
+++ b/librarySDK
@@ -1 +1 @@
-Subproject commit a2b94d4a8a668b9c2bc0f7f253d12beddb1451f0
+Subproject commit a2b94d4a8a668b9c2bc0f7f253d12beddb1451f0-dirty
diff --git a/one.txt b/one.txt
index e69de29..17475fe 100644
--- a/one.txt
+++ b/one.txt
@@ -0,0 +1 @@
+sample change
\ No newline at end of file
diff --git a/settings.patch b/settings.patch
deleted file mode 100644
index d621cdf..0000000
Binary files a/settings.patch and /dev/null differ
